pkg/proxy/winkernel/proxier.go

```diff
@@ -454,7 +443,7 @@ func newSourceVIP(hns HostNetworkService, network string, ip string, mac string,
 }
 
 func (ep *endpointsInfo) Cleanup() {
-	Log(ep, "Endpoint Cleanup", 3)
+	klog.V(3).InfoS("Endpoint cleanup", "spewConfig", spewSdump(ep))
```

Comment:
What spewSdump function does? We should always prefer to pass original struct and depend on logging library sanitization.

Reply:
```
func spewSdump(v interface{}) string {
	scs := spew.NewDefaultConfig()
	scs.DisableMethods = true
	return scs.Sdump(v)
}
```
I don't think logging lib can handle this... Maybe I can move this logic to `Cleanup` function?😳

Reply:
Or just show the `endpointsInfo`😳

Reply:
This code still doesn't show what spew does. Does it just marshall? Can you provide comparison of log that use spew and if we just skip it?

Reply:
A small test:
```
klog.InfoS("Endpoint cleanup", "spewConfig", spewSdump(p))
***the log result:***
I0921 21:10:52.783833   45175 main.go:51] "Endpoint cleanup" spewConfig="(*main.endpointsInfo)(0xc0000be2a0)({\n ip: (string) (len=9) \"127.0.0.1\",\n port: (uint16) 3307,\n isLocal: (bool) false,\n macAddress: (string) (len=5) \"A1:B2\",\n hnsID: (string) (len=3) \"333\",\n refCount: (*uint16)(<nil>),\n providerAddress: (string) (len=7) \"0.0.0.0\",\n ready: (bool) false,\n serving: (bool) false,\n terminating: (bool) false\n})\n"

klog.InfoS("Endpoint cleanup", "spewConfig", p)
***the log result:***
I0921 21:10:52.784251   45175 main.go:52] "Endpoint cleanup" spewConfig=&{ip:127.0.0.1 port:3307 isLocal:false macAddress:A1:B2 hnsID:333 refCount:<nil> providerAddress:0.0.0.0 ready:false serving:false terminating:false}
```
Well, it seems that just a little difference.🙈

Reply:
In that case, please remove spew

Reply:
I see that `proxier.go` have other klog call `spewSdump(v interface{})`, should I remove them all?

Reply:
That would be preferred.